server/tserver/src/main/java/org/apache/accumulo/tserver/TabletIteratorEnvironment.java

```diff
@@ -50,14 +51,16 @@
   private SamplerConfiguration samplerConfig;
   private boolean enableSampleForDeepCopy;
 
-  public TabletIteratorEnvironment(IteratorScope scope, AccumuloConfiguration config) {
+  public TabletIteratorEnvironment(IteratorScope scope, AccumuloConfiguration config,
+      boolean userCompaction) {
```

Comment:
Boolean parameters are notoriously buggy. It'd be better to use an enum instead of a boolean.

Reply:
Added changes, although they don't smell right, might need a new Enum?